styles/learn-x-in-y-minutes/catppuccin.user.css

```diff
@@ -0,0 +1,175 @@
+/* ==UserStyle==
+@name Learn X in Y Minutes Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/learn-x-in-y-minutes
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/learn-x-in-y-minutes
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/learn-x-in-y-minutes/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Alearn-x-in-y-minutes
+@description Soothing pastel theme for Learn X in Y Minutes
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain("learnxinyminutes.com") {
+  @import url("https://unpkg.com/@catppuccin/highlightjs@0.2.2/css/catppuccin.variables.important.css");
+
+  html.light {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  html.dark {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
```

Comment:
the 404 page has no html el with the light or dark class so it doesnt get added only. I think you would have to add some way to detect its the 404 page or add global styling im not sure.